Pipfile

```diff
@@ -0,0 +1,33 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+Jinja2 = "*"
+msgpack-python = ">0.3,!=0.5.5"
+PyYAML = "*"
+MarkupSafe = "*"
+requests = ">=1.0.0"
+tornado = ">=4.2.1,<5.0"
+# Required by Tornado to handle threads stuff.
+futures = {version = ">=2.0", markers = "python_version < '3.0'"}
+pycrypto = ">=2.6.1"
+pyzmq = ">=2.2.0"
+
+[dev-packages]
+mock = ">=2.0.0"
+apache-libcloud = ">=0.14.0"
+boto = ">=2.32.1"
+boto3 = ">=1.2.1"
+moto = ">=0.3.6"
+SaltPyLint = ">=v2017.3.6"
+pytest = ">=3.5.0"
+Pytest-catchlog = {git = "git://github.com/eisensheng/pytest-catchlog.git", ref = "develop", markers = "python_version < '3.0'"}
```

Comment:
Drop this one, if it `py.test` explodes, I have a fix

Reply:
Done